src/global/styles/normalize.scss

```diff
@@ -230,3 +230,46 @@ Add the correct display in Chrome and Safari.
 summary {
   display: list-item;
 }
+
+/*
+Add global styles for the origin of autocomplete dropdown
+TODO: Check if this needs to me moved on another file
+*/
+*[data-autocomplete-origin] {
```

Comment:
Probably, this is not the best place where this CSS should be.
These rules achieve the "wrapping" effect of the overlay around the origin element and, to work, they need to be "global".
Do you have suggestions on where to put them?